inductiva/simulators/simulator.py

```diff
@@ -80,6 +94,8 @@ def run(
 
         self.validate_computational_resources(on)
 
+        kwargs = self._commands_to_json(kwargs)
```

Comment:
In terms of structure, I think we should add a function named `_process_kwargs` where the `_commands_to_json` is used. Otherwise, this seems weird. In case of need, that function can be used for other kwargs in the future.

Reply:
Maybe, but it's tickling me that we are passing all kwargs through a `_commands_to_json` method.

Reply:
I like that!